Automattic/wc-calypso-bridge
# Fix nav unification causing header overflow

Fixes https://github.com/woocommerce/woocommerce-admin/issues/7035.

Nav unification caused WooCommerce header to overflow due to the width rule hardcoded with assumed navigation width. This PR fixes the issue for both business and e-commerce plan user.

### Testing instructions

1. In order to enable nav unification, make sure to have `wpcomsh` and Jetpack installed and connected.
2. If you're experiencing fatal error due to `wpcomsh` conflict, try using my [fork of the bridge helper](https://gist.github.com/ilyasfoo/3a898d3aca8c6a27acc851a28636332d).
2. Make sure new navigation feature is turned off.
3. Navigate to any WooCommerce page, observe a normal header.

### Screenshot:
Before:

![image](https://user-images.githubusercontent.com/3747241/119776851-06316780-bef8-11eb-930f-09a09aa4e7bf.png)

After:

![image](https://user-images.githubusercontent.com/3747241/119776972-2f51f800-bef8-11eb-98da-2c7c61f84989.png)
